pmm-app/tests/verifyQAN_test.js

```diff
@@ -121,3 +122,105 @@ Scenario('Verify Main Metric change reflects in URL @not-pr-pipeline', async (I,
   qanPage.changeMetricTo(metricToReplace, newMetricName);
   qanPage.verifyURLContains(qanPage.urlParts.lockTime);
 });
+
+// TODO: Uncomment after new QAN will be merged
+xScenario(
+  'PMM-T175 - Verify user is able to apply filter that has dots in label @not-pr-pipeline',
+  async (I, qanPage) => {
+    const serviceName = 'ps_5.7_0.0.0.0_1';
+    qanPage.waitForNewQANPageLoaded();
+    const countBefore = await qanPage.getCountOfItems();
+    qanPage.applyFilterNewQAN(serviceName);
+    I.seeInCurrentUrl('service_name=' + serviceName);
+    I.waitForInvisible(qanPage.fields.newQANSpinnerLocator, 30);
+    const countAfter = await qanPage.getCountOfItems();
+    qanPage.verifyChangedCount(countBefore, countAfter);
+  }
+);
+
+// TODO: Uncomment after new QAN will be merged
+xScenario(
+  'PMM-T172 - Verify that selecting a filter updates the table data and URL  @not-pr-pipeline',
+  async (I, qanPage) => {
+    const environmentName = 'ps-dev';
+    qanPage.waitForNewQANPageLoaded();
+    const countBefore = await qanPage.getCountOfItems();
+    qanPage.applyFilterNewQAN(environmentName);
+    I.seeInCurrentUrl('environment=' + environmentName);
+    I.waitForInvisible(qanPage.fields.newQANSpinnerLocator, 30);
+    const countAfter = await qanPage.getCountOfItems();
+    qanPage.verifyChangedCount(countBefore, countAfter);
+  }
+);
+
+// TODO: Uncomment after new QAN will be merged
+xScenario('PMM-T126 - Verify user is able to Reset All filters @not-pr-pipeline', async (I, qanPage) => {
+  const service_name = 'ps_5.7_0.0.0.0_1';
```

Comment:
Ivana I understand why you use this here, but I am sure when we change the default version to ps 8.x this will no longer be valid and the test will start failing, do you think you could make this more generic, maybe passing just `ps_` and trying to apply service name filter?

Reply:
I changed that for this scenario. But I cannot change it for scenario PMM-T175. It's checking the dots int the URL. Look at the test case: https://jira.percona.com/secure/Tests.jspa#/testCase/360.